joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -41,20 +42,30 @@ trait AstCreatorHelper {
 
   private def notHandledText(node: BabelNodeInfo): String =
     s"""Node type '${node.node.toString}' not handled yet!
-       |  Code: '${node.code}'
+       |  Code: '${shortenCode(node.code, length = 50)}'
        |  File: '${parserResult.fullPath}'
        |  Line: ${node.lineNumber.getOrElse(-1)}
        |  Column: ${node.columnNumber.getOrElse(-1)}
        |  """.stripMargin
 
-  protected def notHandledYet(node: BabelNodeInfo): Ast = {
-    val text = notHandledText(node)
+  protected def notHandledYet(node: BabelNodeInfo, additionalInfo: String = ""): Ast = {
+    val text = if (additionalInfo.isEmpty) {
```

Comment:
if it's only one expr in each branch, maybe ditch the brackets?

Reply:
Did it that way everywhere already and its also the default recommendation by various linters.